tests/conftest.py

```diff
@@ -0,0 +1,18 @@
+
+
+from executing._pytest_utils import is_pytest_compatible
+import _pytest.assertion.rewrite as rewrite
+
+
+# this is not an official api, but it is the only way I found to disable pytest rewriting conditionally
+
+original_bailout = rewrite.AssertionRewritingHook._early_rewrite_bailout
```

Comment:
Patching `find_spec` should be much more reliable and understandable since it's an abstract method in Python's import system.